### PR TITLE
New Town Crier System

### PR DESCRIPTION
**Pull request by kevin-10**
- Added non-repeatable quests
- Updated mondain quest system for flexibility in new Quests

### CODE REVIEW

File: Scripts/Services/Town Cryer/Quests/AVisitToCastleBlackthornQuest.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        public override object Description { get { return 1158198; } }
+
+        /* You decide against accepting the quest. */
+        public override object Refuse { get { return 1158130; } }
+
+        /* Visit Castle Blackthorn in Northern Britain. */
+        public override object Uncomplete { get { return 1158199; } }
+
+        /* You have braved the wilds of Britannia and slayed a mighty beast! You have meticulously documented your kill 
+         * and submitted it to the Ranger's Guild for evaluation. If luck was on your side you may indeed have
```
Comment (Tasanar) [inline Scripts/Services/Town Cryer/Quests/AVisitToCastleBlackthornQuest.cs]: This message seems to be for the Huntmaster quest and not blackthorn.

Comment (kevin-10) [reply Scripts/Services/Town Cryer/Quests/AVisitToCastleBlackthornQuest.cs]: The only quest i was able to complete was the huntsmaster, and i got a deed. ALso, that cliloc is correct, just the wrong discription which doesnt matter but ill update it.

**Comment by Tasanar** [comment]
I have personally done Exploring the deep. That quest 100% grants the title and no deed. Only the character that completes the quest gets that. 

Other than hunt master I could not find any of those title deeds for sale on Vendor Search, on Atlantic.

**Comment by kevin-10** [comment]
Also, got a deed doing the britannia castle quest.  Did you do the exploring the deep as the Town Crier quest?  Its been noted in stratics that each quest gives out a title deed.  That may have changed since you did it last.

**Pull request merged**